### PR TITLE
ship to production

### PR DESCRIPTION
**Pull request by PierreJeanjacquot**

### CODE REVIEW

File: documentation/tools/oracle-factory/quick-start.md [hunk 1]
```diff
@@ -1,174 +1,88 @@
...
-### **3.2. Create, Update, and Read Oracle**
+Import and initialize the Oracle Factory SDK in your application.
 
-Utilize the SDK's methods to create, update, and read from the oracle.
+{% tabs %}
+{% tab title="Browser" %}
+Instantiate the SDK in your front-end project:
 
-#### **3.2.1. Creating an Oracle**
+#### 2.4.1. Basic Instantiation
```
Comment (cedric25) [inline documentation/tools/oracle-factory/quick-start.md:63]:
```suggestion
#### 1.4.1. Basic Instantiation
```

File: documentation/tools/oracle-factory/quick-start.md [hunk 2]
```diff
@@ -1,174 +1,88 @@
...
-  }
-};
-
-export default updateOraclePromise;
-```
+{% tab title="NodeJS" %}
+Instantiate the SDK in your back-end project:
 
-#### **3.2.3. Reading from an Oracle**
+#### 2.4.1. Basic Instantiation
```
Comment (cedric25) [inline documentation/tools/oracle-factory/quick-start.md:77]:
```suggestion
#### 1.4.1. Basic Instantiation
```

File: documentation/tools/oracle-factory/quick-start.md [hunk 3]
```diff
@@ -1,174 +1,88 @@
...
 
-### **1.1. Prerequisites**
+Before getting started, ensure that you have the following installed on your system:
 
-Before starting, make sure you have the following installed:
+\- [**Node.js**](https://nodejs.org/en/) version 18 or higher
 
-- [**Node.js**](https://nodejs.org/en/) version 18 or higher
-- [**NPM**](https://docs.npmjs.com/) (Node.js package manager)
+\- [**NPM**](https://docs.npmjs.com/) (Node.js package manager)\
```
Comment (cedric25) [inline documentation/tools/oracle-factory/quick-start.md:11]:
```suggestion
\- [**NPM**](https://docs.npmjs.com/) (Node.js package manager)
```

**Comment by PierreJeanjacquot** [comment]
set as draft need changes on OF part

File: documentation/tools/oracle-factory/quick-start.md (hunk 3 above)
Comment (PierreJeanjacquot) [reply documentation/tools/oracle-factory/quick-start.md:11]: addressed in https://github.com/iExecBlockchainComputing/documentation-tools/pull/27

File: documentation/tools/oracle-factory/quick-start.md (hunk 2 above)
Comment (PierreJeanjacquot) [reply documentation/tools/oracle-factory/quick-start.md:77]: addressed in https://github.com/iExecBlockchainComputing/documentation-tools/pull/27

File: documentation/tools/oracle-factory/quick-start.md [hunk 4]
```diff
@@ -1,174 +1,88 @@
...
 
-## 2. Initialize Your Project
+### **1.2. Start a new project**
 
-### **2.1. Clone the Project**
+In this section, we will show you how to set up and call the different methods of the SDK in a React.js app.
```
[the remark this answers is not in the document]
Comment (PierreJeanjacquot) [reply documentation/tools/oracle-factory/quick-start.md:15]: addressed in https://github.com/iExecBlockchainComputing/documentation-tools/pull/27

File: documentation/tools/dataprotector/methods/transferownership.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
+# transferOwnership
+
+Allows transferring ownership of protected data to a new owner, identified by their ETH address. The return value provides a transaction hash and confirmation of the new owner of the data.
```
Comment (cedric25) [inline documentation/tools/dataprotector/methods/transferownership.md:3]: All other method descriptions start with "Method to ..."

```suggestion
Method to transfer ownership of a protected data to a new owner, identified by their ETH address. The return value provides a transaction hash and confirmation of the new owner of the data.
```

File: documentation/tools/dataprotector/methods/transferownership.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    newOwner: '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
+});
+```
+
+## Return value example
+
+```javascript
+[
+    {
+        "address": "0xA0Cf798816D4b9b9866b5330EEa46a18382f251e",
```
Comment (cedric25) [inline documentation/tools/dataprotector/methods/transferownership.md:19]: When calling the method: `protectedData`
When getting the response, naming becomes `address`?

Maybe keep the property name as `protectedData` also in the response?

**Review by cedric25 — APPROVED** [review]

**Pull request merged**